src/background.js

```diff
@@ -41,26 +41,25 @@ function hexStringToByte(str) {
     return new Uint8Array();
   }
 
-  var a = [];
-  for (var i = 0, len = str.length; i < len; i+=2) {
+  const a = [];
+  for (let i = 0, len = str.length; i < len; i+=2) {
     a.push(parseInt(str.substr(i,2),16));
   }
 
   return new Uint8Array(a);
 }
 
 function extractAccessToken(redirectUri) {
-  let m = redirectUri.match(/[#\?](.*)/);
-  if (!m || m.length < 1)
-    return null;
-  let params = new URLSearchParams(m[1].split('#')[0]);
+  const m = redirectUri.match(/[#?](.*)/);
+  if (!m || m.length < 1) return null;
+  const params = new URLSearchParams(m[1].split('#')[0]);
   return params.get('code');
 }
 
 function getKintoFxAConfig(kinto_url) {
-  return fetch(KINTO_URL+'/fxa-oauth/params')
+  return fetch(kinto_url + '/fxa-oauth/params')
```

Comment:
Converted this to `kinto_url` argument since we pass the global `const` value below on L122 anyways.